client/src/ForgotPassword.tsx

```diff
@@ -0,0 +1,140 @@
+  const xPadding = 12;
+  const yPadding = 6;
+  const yMargin = 8;
+
+  return {
+    paper: {
+      maxWidth: 821 - theme.spacing(xPadding),
+      maxHeight: 732 - theme.spacing(yPadding),
+      borderRadius: 10,
+      marginTop: theme.spacing(yMargin),
+      marginBottom: theme.spacing(yMargin),
+      paddingTop: theme.spacing(yPadding),
+      paddingBottom: theme.spacing(yPadding),
+      paddingLeft: theme.spacing(xPadding),
+      paddingRight: theme.spacing(xPadding),
+      margin: 'auto',
+    },
+    header: { fontWeight: 'bold', marginBottom: 68 },
+    button: {
+      borderRadius: 0,
+      height: 62,
+      textTransform: 'none',
+    },
+  };
+});
+
+interface UserLoginData {
+  email: string;
+}
+
+const initialFormData: UserLoginData = {
+  email: '',
+};
+
+interface Error {
+  type: '' | 'email';
+  message: string;
+}
+
+function ForgotPassword() {
+  const classes = useStyles();
+  //const history = useHistory();
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [error] = React.useState<Error | null>(null);
+  const [showSnackbar, setShowSnackbar] = React.useState<boolean>(false);
+  //const [, setUser] = React.useContext(UserContext);
+
+  const [formData, setFormData] = React.useState(initialFormData);
+  const instructions =
+    "Enter your email and we'll send you a link to reset you password if you have an account.";
+  const emailResponse =
+    'If this email exists as a user you will be sent an email to reset your password.';
+
+  const generateForm = () => {
+    return (
+      <Grid container item xs={12}>
+        <form onSubmit={handleSubmit} style={{ width: '100%' }}>
+          <EmailInput
+            value={formData.email}
+            placeholder="jane@nonprofit.com"
+            onChange={handleChange}
+            error={error?.type === 'email' ? error.message : null}
+          />
+          <Button
+            className={classes.button}
+            style={{ backgroundColor: '#C4C4C4', color: 'white' }}
+            fullWidth
+            type="submit"
+          >
+            Send Email To Set New Password
+          </Button>
+          {/* Placeholder for loading  - waiting on UI/UX response as to what they want. */}
+          {isLoading && <Typography>Loading</Typography>}
+        </form>
+      </Grid>
+    );
+  };
+
+  const handleChange = (evt: React.ChangeEvent<HTMLInputElement>): void => {
+    const { name, value }: { name: string; value: string } = evt.target;
+    setFormData((fData) => ({
+      ...fData,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = async (evt: React.FormEvent): Promise<void> => {
+    evt.preventDefault();
+    setIsLoading(true);
+
+    await fetch('http://localhost:3001/api/users/reset_password', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(formData),
+    });
+
+    setIsLoading(false);
+    setShowSnackbar(true);
```

Comment:
also clear the formData in here

Reply:
I think I forgot to do this because I don't generate the form after you submit, but I added some code to clear it.